kakarot_scripts/deploy_kakarot.py

```diff
@@ -23,6 +23,7 @@
     invoke,
     upgrade,
 )
+from tests.utils.constants import DEFAULT_GAS_PRICE
```

Comment:
`from kakarot_scripts.constants`

prod files should not import from test files, the other way is possible